src/jshint.js

```diff
@@ -2218,6 +2217,19 @@ var JSHINT = (function() {
     return this;
   }));
   prefix("new", function() {
+    var mp = metaProperty("target");
```

Comment:
If `metaProperty` only returns if value === "target", JSHint would throw a ton of errors when linting invalid meta properties because it would treat `new` as a construction.